mariadb_skysql/README.md

```diff
@@ -0,0 +1,64 @@
+# Agent Check: MariaDB SkySQL
+
+## Overview
+
+This check monitors [MariaDB SkySQL][1]. SkySQL is a multi-cloud database-as-a-service offering from MariaDB Plc.
+
+The MariaDB SkySQL integration allows full monitoring of all your MariaDB SkySQL server deployments: Enterprise Server (Single and Replicated), Xpand, Columnstore, as well as MaxScale. Database administrators can track system resource metrics, such as memory, CPU, disk, and network utilization in addition to database specific telemetry such as query performance, connections, threads, cluster, replication status, and many others.
+
+
+
+## Setup
+
+### Installation
+
+To install the MariaDB SkySQL check on your host:
+
+1. Install the [developer toolkit][10] on any machine.
+
+2. Run `ddev release build mariadb_skysql` to build the package.
+
+3. [Download the Datadog Agent][2].
+
+4. Upload the build artifact to any host with an Agent and
+ run `datadog-agent integration install -t datadog-mariadb_skysql==0.0.1`
+ This command assumes that the build directory is in ./integration-extras/mariadb_skysql/dist. You may need to modify this path to match your agent environment settings.
```

Comment:
I believe the customer installing this would just have to run a simple install command, like:

`datadog-agent integration install -t datadog-mariadb-skysql==1.0.0`

@alopezz can you confirm?

Reply:
Indeed, see https://docs.datadoghq.com/agent/guide/use-community-integrations/?tab=agentv721v621#setup